blockchain/blockchain.go

```diff
@@ -696,7 +696,7 @@ func (bc *blockchain) commitBlock(blk *Block) error {
 	if bc.sf != nil {
 		ExecuteContracts(blk, bc)
 		if err := bc.sf.CommitStateChanges(blk.Height(), blk.Transfers, blk.Votes, blk.Executions); err != nil {
-			return err
+			logger.Fatal().Err(err).Msgf("Failed to commit state changes on height %d", blk.Height())
```

Comment:
Leave a  TODO message to say we will fix the log level in the future, once committing the block and the state become a transaction